historical/test/commands-cyBypassUI.js

```diff
@@ -205,8 +205,7 @@ Cypress.Commands.add('giCreateGroup', (name, {
     cy.getByDT('nextBtn').click()
 
     cy.getByDT('rulesStep').within(() => {
-      const threshold = ruleName === 'percentage' ? ruleThreshold * 100 : ruleThreshold
-      cy.getByDT(ruleName, 'label').click()
```

Comment:
Please allow the historical files to preserve history :P
(They should not be modified)

Reply:
done it in this [commit](https://github.com/okTurtles/group-income/pull/1407/commits/1478f02b565169320ea860dc549665f64aff2401)